File: ios/BFExplorer/Plaoc/IPCCore/IPC/IpcStreamAbort.swift
//
//  IpcStreamAbort.swift
//  IPC
//
//  Created by ui03 on 2023/2/13.
//

import UIKit

struct IpcStreamAbort {

    var type: IPC_MESSAGE_TYPE = .STREAM_ABORT
    private(set) var stream_id: String?
    
    init(stream_id: String) {
        self.stream_id = stream_id
    }
}

extension IpcStreamAbort: IpcMessage {
    
    init() {
        
    }
}

File: desktop-dev/src/core/ipc/IpcStreamAbort.ts
import { IPC_MESSAGE_TYPE, IpcMessage } from "./const.ts";

export class IpcStreamAbort extends IpcMessage<IPC_MESSAGE_TYPE.STREAM_ABORT> {
  constructor(readonly stream_id: string) {
    super(IPC_MESSAGE_TYPE.STREAM_ABORT);
  }
}
